lib/styles/settings/_icons.scss

```diff
@@ -8,3 +8,5 @@ $icon-xl: 48px;
 $icon-xxl: 64px;
 
 $min-touchable-size: 40px;
+
+$animation-spin: a-fadein 300ms ease-in-out 0s 1, a-spinAround 1s infinite linear; /* stylelint-disable-line value-keyword-case */
```

Comment:
I'm not a fan of mixing these two animations into one variable. Expecially when variable name doesn't suggest that

Reply:
Sure.